templates/layout.tmpl

```diff
@@ -35,7 +35,7 @@
                     <div class="col col--md-40 col--lg-44">
                         <span class="banner__text-icon">Beta</span>
                         <div class="banner__body">
-                            <p class="banner__text">This is a new way of getting data - your <a href="/feedback?service=dev">feedback</a> will help
+                            <p class="banner__text">This is a new way of getting data - your <a href="https://beta.ons.gov.uk/feedback?service=dev">feedback</a> will help
```

Comment:
Although `beta.ons.gov.uk` will work right now due to the redirect, it would be better to use `www.ons.gov.uk` as this is more future proof.

Reply:
Thank you, just updated